### PR TITLE
READ Implemented

### PR DESCRIPTION
**Pull request by danicrosby**
Planes
Baggage
Crews
Food Servies
Airports

### CODE REVIEW

File: package.json
Comment (TrinityChristiana) [inline package.json]: This needs to be uninstalled. 
```suggestion
```

File: documentation/README.md
Comment (TrinityChristiana) [inline documentation/README.md:75]: I see that someone's looking out for `console.warns`!!! I love it!

File: src/javascripts/components/airports.js
Comment (TrinityChristiana) [inline src/javascripts/components/airports.js:12]: Looks like your edit buttons and delete buttons are showing up even if you're logged out. We want them to only show up if you're logged in. We can add this functionality later. Create a ticket to pull in at the end of this sprint and add a link to it here as a response! If you already have a ticket for this work you can add that here instead!

File: src/javascripts/views/startApp.js
Comment (TrinityChristiana) [inline src/javascripts/views/startApp.js]: Do you plan on using this commented-out code later?

File: package.json
Comment (BenJarrett) [reply package.json]: We're reading through #98, and we saw that Nathan has 14.15.4 of node. The issue ticket says that Honey Rae had to downgrade to 14. But it looks like Nathan is already on 14? Am I interpreting this wrong?

File: src/javascripts/views/startApp.js
Comment (BenJarrett) [reply src/javascripts/views/startApp.js]: Yeah, we will be using this code later. We were just commenting it out to make things work while merging locally

Comment (TrinityChristiana) [reply src/javascripts/views/startApp.js]: 😎 Just making sure!

Comment (BenJarrett) [reply src/javascripts/views/startApp.js]: Np! Thanks for checking!

File: documentation/README.md
Comment (danicrosby) [reply documentation/README.md:75]: Webpack instructions ;)

File: src/javascripts/components/airports.js
Comment (danicrosby) [reply src/javascripts/components/airports.js:12]: We have a workaround in the next pull. I'll make a note and double-check this is what is needed.

Comment (TrinityChristiana) [reply src/javascripts/components/airports.js:12]: Yeah! Can you make an issue to track that work? Also so that I can remember to check for it in the next Milestone approval.

**Comment by TrinityChristiana** [comment]
I also need a deployed link to test out these changes before this can be approved.

File: package.json
Comment (TrinityChristiana) [reply package.json]: You're not interpreting this wrong. We're continuing this conversation in the DIscussion Ticket [here](https://github.com/nss-evening-cohort-14/evening-client-side/discussions/194) if you wanted to follow along.

**Comment by danicrosby** [comment]
https://nss-pan-am-2021.netlify.app/

@TrinityChristiana : I created a new Netlify Account hoping that will solve our issues. I ran the dev branch in Netlify and it has updated code. We have a couple placement code bugs that will be fixed with the CREATE  PR's

**Comment by danicrosby** [comment]
@TrinityChristiana : New Netlify Deployment Link: https://nss-pan-am-2021.netlify.app/

**Comment by TrinityChristiana** [comment]
Just looked at the deployed link and all I can see is the Food Service View. If I click on the Navbar items, the view does not change. I also cannot view items if logged out. 

For the logged-out view, we can create another ticket to pull in at the end of the sprint.

**Comment by danicrosby** [comment]
@TrinityChristiana , Yes, Sara has an updated NavBar that will toggle through the individual categories. And I have an updated HOME page. We were going to push that up and merge with team on our CREATE portion today. I will make an issue ticket on the LOGOUT VIEW now.

Sara and I can push those things up with this ticket, but we have already finished our CREATE to keep the ball moving. We can push our work that included if that's ok with you.

**Comment by TrinityChristiana** [comment]
This is for approval of the READ milestone which means I should be able to read the following items
- Planes
- Baggage
- Crews
- Food Servies
- Airports
Currently, I can only read the Food Services. So for this PR to get approved, the navbar has to work. So those changes need to be in this PR. 

FYI: I don't want to see any of the new create changes in this PR. You all really are not supposed to be working on the next milestone until you get approval to move on. I won't approve this PR if I see any of the create changes.

**Comment by danicrosby** [comment]
@TrinityChristiana : we are ready!  https://nss-pan-am-2021.netlify.app/

**Review by TrinityChristiana — APPROVED** [review]
That's what I'm talking about!!! Good job everyone!!!

**Pull request merged**
